openapi.yaml

```diff
@@ -268,39 +249,48 @@ components:
       description: Unique identifier for museum ticket. Generated when purchased.
       type: string
       format: uuid
+      readOnly: true
       example: a54a57ca-36f8-421b-a6b4-2e8f26858a4c
     TicketConfirmation:
       description: Unique confirmation code used to verify ticket purchase.
       type: string
       example: 'ticket-event-a98c8f-7eb12'
-    BuyMuseumTicketsResponse:
-      description: Details for a museum ticket after a successful purchase.
+    Ticket:
+      description: Ticket for museum entry, can be general admission or special event.
       type: object
       properties:
-        message:
-          $ref: "#/components/schemas/TicketMessage"
-        eventName:
-          $ref: "#/components/schemas/EventName"
         ticketId:
           $ref: "#/components/schemas/TicketId"
-        ticketType:
-          $ref: "#/components/schemas/TicketType"
         ticketDate:
           description: Date the ticket is valid for.
           $ref: "#/components/schemas/Date"
-        confirmationCode:
-          $ref: "#/components/schemas/TicketConfirmation"
+        ticketType:
+          $ref: "#/components/schemas/TicketType"
+        eventId:
+          description: Unique identifier for a special event. Required if purchasing tickets for the museum's special events.
```

Comment:
not sure if there should be additional property next to `$ref`, I think it's cut off, so the description from `EventId` will be used if rendered with any tools

Reply:
I believe this is the correct format since OpenAPI 3.1 supports `description` fields alongside schema refs. See the specification https://spec.openapis.org/oas/v3.1.0#reference-object